src/MongoDB.Driver/Linq/Processors/SerializationBinder.cs

```diff
@@ -151,6 +159,12 @@ protected override Expression VisitMember(MemberExpression node)
                                 mex);
                         }
                     }
+
+                    var rawParameterExpression = (node.Expression as ParameterExpression);
```

Comment:
Weren't we going to take "raw" out of the variable name?

Reply:
I've answered you in PR for the ticket 2269. But Ok, I will delete this part.